### ISSUE TITLE
Add French translation

### ISSUE THREAD

**Issue by vlevieux** [flaskcwg/translation-coordination#5]
Language:

## Initial tasks
- [x] Create an empty repository on GitHub. Name the repository as `flask-docs-<lang code>`, for example, `flask-docs-es`.
- [x] Set up your repository based on the translation template repository:
```
$ git clone https://github.com/flaskcwg/translation-template.git flask-docs-<lang code>
$ cd flask-docs-<lang code>
$ git remote set-url origin <the-Git-URL-to-your-repository>
$ git remote add upstream https://github.com/pallets/flask.git
$ git fetch upstream
$ git merge upstream/2.0.x
$ git push -u origin main
```
- [x] Create a virtual environment, then install the requirements:
```
$ python -m pip install --upgrade pip setuptools
$ pip install -r requirements/dev.txt
$ pip install -e .
$ pre-commit install
```
- [x] Generate the `.pot` and `.po` files (replace `<lang code>` with your language code):
```
$ cd docs
$ make gettext  # use ".\make.bat gettext" on Windows
$ sphinx-intl update -p _build/gettext -l <lang code>
```
- [x] Replace the `<LANG>` in the `README.md`, the `language` config, and the
`html_search_language` config in `docs/conf.py` with your language code.
Update the `.po` file examples.
- [x] Setup ReadtheDocs to connect your repository, use `flask-<lang code>` as the subdomain, set the default version to the `main` branch, and enable the "single version" mode.
- [ ] Translate the `README.md` and the pull request template (optional).
- [x] Update the title, and all the `<UPDATE THIS>` placeholders  in `README.md` and `docs/conf.py`. You can update the `README.md` to add more format or translate tips for your language.
- [ ] You can choose to use [Transifex](https://www.sphinx-doc.org/en/master/usage/advanced/intl.html#using-transifex-service-for-team-translation) or other translation platforms. In that case, you will need to rewrite the "Contributing Guide" in the `README.md`.
- [x] Update the `Language-Team` value in all `.po` files.
- [x] Translate `html_title` and project links in the `docs/conf.py` file.
- [x] Translate the `docs/index` file (you can make a localized logo for the index page).
- [x] Translate the `docs/foreword` file.
- [x] Translate the `docs/advanced-foreword` file.
- [x] Translate the `docs/installation` file.
- [x ] Translate the `docs/quickstart` file.
- [x] Translate the "Tutorial" part of the docs.
- [ ] Switch the repository into the flaskcwg organization.

## Public

Tasks after the repository is switched into flaskcwg:

- [ ] Add a new language entry to [the README](https://github.com/flaskcwg/translation-coordination/blob/main/README.md).
- [ ] Add a new language entry to [translations page](https://github.com/flaskcwg/flaskcwg.github.io/blob/source/templates/translations.html).

## Maintenance

Tasks when the whole documentation is translated:

- [ ] Create a 2.0.x branch, set it as default version on ReadtheDocs.
- [ ] Rewrite the contributing guide in the `README.md` of the translation repository, encourage people to fetch upstream and update docs (see details below).
- [ ] Rename the section "Translation To-do List" to "Translators". You can also add
a "Reviewers" section.
- [ ] Switch the docs into http://flask.palletsprojects.com/.

You should sync documentation updates from Flask periodically:

```
$ git fetch upstream
$ git merge upstream/2.0.x
$ cd docs
$ make gettext  # use ".\make.bat gettext" on Windows
$ sphinx-intl update -p _build/gettext
```

Then update the translation for new changes.

**Comment by Abdur-rahmaanJ** [comment]
@Jalkhov can you follow the issue about migration. Adding you @vlevieux as the french coordinator!

**Comment by Jalkhov** [comment]
@vlevieux Do you have discord?

**Comment by vlevieux** [comment]
@Jalkhov Yes I do, I already joined the Pallets Projects discord. My ID: @Reese41#6396

**Comment by vlevieux** [comment]
@Abdur-rahmaanJ @Jalkhov Unfortunately I can't transfer the ownership to flaskcwg. I don't have the permissions to create a public repo on flaskcwg.

**Comment by Abdur-rahmaanJ** [comment]
@vlevieux Now supposed to be ok. Privileges granted.
